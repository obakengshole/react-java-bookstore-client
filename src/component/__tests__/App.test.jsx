import React from 'react'
import renderWithRedux from '../../util/testUtil'
import App from '../App'

describe('App component', () => {
    test ('should render app without error', () => {
        const { asFragment } = renderWithRedux(<App />, {})
        expect(asFragment()).toMatchSnapshot()
    })
})