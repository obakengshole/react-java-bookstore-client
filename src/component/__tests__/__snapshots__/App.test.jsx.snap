// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component should render app without error 1`] = `
<DocumentFragment>
  <div
    class="MuiBox-root MuiBox-root-1"
  >
    <div
      class="MuiBox-root MuiBox-root-2"
    >
      <header
        class="MuiPaper-root MuiAppBar-root MuiAppBar-positionFixed MuiAppBar-colorPrimary mui-fixed MuiPaper-elevation4"
        posisiton="fixed"
      >
        <div
          class="MuiToolbar-root MuiToolbar-dense MuiToolbar-gutters"
        >
          <h6
            class="MuiTypography-root MuiTypography-h6 MuiTypography-colorInherit"
          >
            Book Store
          </h6>
        </div>
      </header>
    </div>
    <div
      class="MuiBox-root MuiBox-root-3"
    >
      <div
        class="MuiBox-root MuiBox-root-8 makeStyles-bookContainer-4"
      >
        <div
          class="MuiBox-root MuiBox-root-9 makeStyles-bookFilter-5"
        >
          <div
            class="MuiPaper-root makeStyles-bookFilterPaper-6 MuiPaper-elevation1 MuiPaper-rounded"
          >
            Book Filter
          </div>
        </div>
        <div
          class="MuiBox-root MuiBox-root-10 makeStyles-bookList-7"
        >
          Display books here
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
